projects/kubernetes-sigs/cluster-api-provider-vsphere/build/create_manifests.sh

```diff
@@ -35,10 +35,14 @@ cd $REPO
 
 yq eval -i -P ".spec.template.spec.containers[0].args += [\"--namespace=eksa-system\"]" config/manager/manager.yaml
 
-make manifests STAGE="release" \
-  MANIFEST_DIR="out" \
-  PULL_POLICY="IfNotPresent" \
-  IMAGE="${IMAGE_REPO}/kubernetes-sigs/cluster-api-provider-vsphere/release/manager:$IMAGE_TAG"
+make manifest-modification STAGE="release" \
```

Comment:
does these still build to and create files which are similiar to what we were producing? probably best to diff what this creates with what we shipped in 0.18.

Reply:
Yeah, they basically split this command and the combination of these two results in the same outputs from what I checked after running the `make build` command. I'll double check with a diff we shipped in 0.18